pages/source-code.html

```diff
@@ -94,8 +94,10 @@ <h4><a href="#uyuni-project">Uyuni Project</a></h4>
             <ul>
                 <li><a href="https://github.com/uyuni-project/uyuni" target="_blank">uyuni</a>: Contains the main Uyuni source code (including, for example the source code for bakend, frontend, database or cucumber testsuite).</li>
                 <li><a href="https://github.com/uyuni-project/retail" target="_blank">retail</a>: Uyuni Retail salt formulas and related tools and documentation</li>
+                <li><a href="https://github.com/SUSE/manager-build-profiles" target="_blank">manager-build-profiles</a>: Sample files: AutoYaST profiles, Kiwi XML definitions for Retail POS images and Docker container definition</li>
```

Comment:
This sectionb is only for the "Uyuni project", meaning the github Uyuni project, so this needs to go on the next section (external git repositories).

Either that, or we move it from SUSE org into uyuni-project org :-)

Reply:
Done